src/components/controls.js

```diff
@@ -155,7 +155,9 @@ export default function (Glide, Components, Events) {
      * @return {Void}
      */
     click (event) {
-      event.preventDefault()
+      if (!supportsPassive && event.type === 'touchstart') {
```

Comment:
The `!supportsPassive` is required for browsers like IE 11 which do not support passive event listeners. Without this check IE 11 would not work as expected and since it is included in the supported browsers list it should work like other browsers.

I am not aware of the issue with two slides moved at once, but that sounds like a separate issue. This pull request is only to fix the Chrome warning.

Reply:
Strange... I have applied this particular modification to my local build (otherwise using `v3.3.0`) and it works fine on Chrome and IE11. I believe you mean desktop IE11 because that's what I've tested with.

Note that the `!supportsPassive` check is only needed when calling `event.preventDefault()` which I don't think is needed. I may be missing something here regarding the inner workings of the library, though, since I'm not 100% familiar with the source.

Reply:
this resolves?

Reply:
@alexalannunes Resolves what?

The developers didn't seem to think this would be important but I would be still interested to hear any actual use case where my solution wouldn't work or why the `!supportsPassive` check is even relevant in this piece of code. My suggestion doesn't call `event.preventDefault()` (because it's not really needed) so I'm not sure why checking the passive events support would be even important.